Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -6044,5 +6045,37 @@ public void ShouldNotDuplicateRequestPathTwiceInCaseOfRecursionGuardException()
 
             // Teardown
         }
+        
+        [Theory]
+        [InlineData(typeof(byte))] 
+        [InlineData(typeof(sbyte))] 
+        [InlineData(typeof(short))]  
+        [InlineData(typeof(ushort))]
+        [InlineData(typeof(int))]  
+        [InlineData(typeof(uint))]  
+        [InlineData(typeof(long))] 
+        [InlineData(typeof(ulong))] 
+        [InlineData(typeof(decimal))]
+        [InlineData(typeof(float))]  
+        [InlineData(typeof(double))]
+        public void Issue897_ShouldCorrectlyHandleRequestsWithSameMinimumAndMaximumValue(Type type)
+        {
+            // Fixture setup
+            var expectedValue = Convert.ChangeType(42, type, CultureInfo.InvariantCulture);
+            var fakeMember = new FakeMemberInfo(
+                new ProvidedAttribute(
+                    new RangeAttribute(type, "42", "42"),
+                    inherited: false
```

Comment:
Perhaps, we could even parameterize the `42` magic number, but not that big of a deal.

Reply:
Yep, that was my original design. I didn't want to hardcode string and wanted to parameterize `42` in it's native type (e.g. `42L`, `42.0`, etc). However, it appeared that you cannot provide a constant for the `decimal` type 😕 Therefore, we should either extract all the values to a `TheoryData<>` member or perform a conversion from string.

I decided to proceed with the latest approach. In that light it doesn't have too much sense to hardcode that string value for each theory case. Therefore, let keep it as is.

If you see a better way - let me know and I'll follow that approach in future.